.github/workflows/pre-merge-to-master.yaml

```
@@ -6,43 +6,63 @@ on:
         uses: actions/checkout@v4
 
-      - name: Setup Environment
-        uses: ./.github/actions/setup-environment
+      - name: Prepare Mod Environment
+        if: ${{ matrix.use_mod_env }}
+        uses: ./.github/actions/setup-mod-env
 
-      - name: Build Solution
-        run: dotnet build -c Release
+      - name: Prepare .NET Environment
+        if: ${{ not matrix.use_mod_env }}
```

actionlint (1.7.8)

parser did not reach end of input after parsing the expression. 3 remaining token(s) in the input: "IDENT", ".", "IDENT"

(expression)